keyboards/keychron/k8/keymaps/iso/keymap.c

```diff
@@ -19,78 +20,73 @@
 #include QMK_KEYBOARD_H
 
 const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
-
-
-/*				+--------------------------------------------------------------------------+----------------+
+/*			+--------------------------------------------------------------------------+----------------+
 				| ESC |  | F1 | F2 | F3 | F4 | | F5 | F6 | F7 | F8 | | F9| F10| F11| F12|  | |PSCR|????|PAUS|
 				+--------------------------------------------------------------------------+------|----|----|
  				|  ~  |  1 |  2 |  3 |  4 |  5 |  6 |  7 |  8 |  9 |  0 |  - |  = | BACKSP | |INS |SCRL|PGUP|
  				+--------------------------------------------------------------------------+------|----|----|
- 				|  TAB  |  Q |  W |  E |  R |  T |  Y |  U |  I |  O |  P |  [ |  ] |   \  | |DEL |END |PGDN|
- 				+--------------------------------------------------------------------------+------|----|----|
- 				| CAPSLCK  |  A |  S |  D |  F |  G |  H |  J |  K |  L | ; | ' |  RETURN  |                |
+        |  TAB  |  Q |  W |  E |  R |  T |  Y |  U |  I |  O |  P |  [ |  ] |      | |DEL |END |PGDN|
+        +--------------------------------------------------------------------  RET | +--------------+
+        | CAPSLCK  |  A |  S |  D |  F |  G |  H |  J |  K |  L | ; | ' |  # |     |                |
```

Comment:
identations should be space, or this magnificent thing happens

Reply:
Darn.. Its vsCode that do some weird auto detection on space / tabs..

Reply:
Should be fixed in keymaps now..